concepticondata/conceptrelations.tsv

```diff
@@ -793,3 +793,4 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 619	ANIMAL	narrower	3923	FEMALE ANIMAL
 1553	MALE	narrower	3924	MALE ANIMAL
 619	ANIMAL	narrower	3924	MALE ANIMAL
+960	WIND	narrower	3982	BREEZE
```

Comment:
Careful with these relations here: We could also say "storm" is narrower than "wind", but we haven't done so so far: https://concepticon.clld.org/parameters/858 I would say, that in these cases it is better to leave concepts as separate entities. We use the narrower-broader relations only when we have clear-cut cases, such as kinship, where we have clear over-under-specification, etc., but avoid to add too many relations on concepts where it is less clear. 

The alternative is to check for wind / storm / tornado and add non-existing relations here. 

Maybe we need a discussion on this?

Reply:
I'd be fine to remove this relation is well, as this seems to be what has done so far.

Reply:
I would also argue against adding the relation. We have too many weather-related concepts, which then also need a thorough check, and I'm not sure that we will find many clear cases.